package.json

```diff
@@ -2,10 +2,11 @@
   "name": "service",
   "version": "0.1.0",
   "description": "Service side of clearlydefined.io.",
+  "main": "./bin/www",
```

Comment:
I'm not so sure of this change, is this something required for this implementation?